__tests__/app.test.js

```diff
@@ -411,3 +411,38 @@ describe("DELETE /api/comments/:comment_id", () => {
     return request(app).delete("/api/comments/not-a-comment_id").expect(400);
   });
 });
+
+describe("GET /api/users", () => {
+  test("200: Responds with an array of all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        users.forEach((user) => {
+          console.log(user);
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("200: Responds with an empty array when there are no users in the database", () => {
+    return db.query(`DELETE FROM comments`).then(() => {
+      return db.query(`DELETE FROM articles`).then(() => {
+        return db.query(`DELETE FROM users`).then(() => {
+          return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({ body }) => {
+              console.log(body);
+              expect(body.users).toEqual([]);
+            });
+        });
+      });
+    });
+  });
```

Comment:
I like the idea of this test, but it isnt needed. We have the data in the database and that is what we are testing is coming back from this request. We shouldnt really ever expect the database to be empty so this behaviour shouldnt need to be tested for